README.md

```diff
@@ -57,268 +51,76 @@ This will build and run a Docker image that will work in the same way as if you
-
-        #!/bin/sh
-        export LAUNCHER_TESTS_SV_SIMULATION=false
-        export LAUNCHER_MISSIONCONTROL_GITLAB_USERNAME=<your gitlab username>
-        export LAUNCHER_MISSIONCONTROL_GITLAB_PRIVATE_TOKEN=<your gitlab token created from above>
-        export LAUNCHER_MISSIONCONTROL_GITLAB_USERNAME=<your bitbucket username>
-        export LAUNCHER_MISSIONCONTROL_GITLAB_PRIVATE_TOKEN=<your bitbucket app password created from above>
-
-
-    Use `source ~/launcher-missioncontrol-env.sh` to make your changes visible; The Launcher environment should be displayed in the terminal.
-
-6. (Optional) Ensure from the previous steps all environment variables are properly set up and sourced into your terminal. You can use [the script defined below](README.md#setting-up-the-environment) to do that for you.
-
-
-
-Run the OSIO addon
-------------------
-See how to run the [fabric8-ui project](https://github.com/fabric8-ui/fabric8-ui) and set the environment variables to either prod, pre-prod or dev.
-Then override the `FABRIC8_FORGE_API_URL` with the instance that you are running like this:
-```
-export FABRIC8_FORGE_API_URL=http://localhost:8080
-```
-Before starting this project set `WIT_URL`, `AUTH_URL`, `KEYCLOAK_SAAS_URL` and `OPENSHIFT_API_URL` for prod this would be:
-```
-export WIT_URL=https://api.openshift.io
-export AUTH_URL=https://auth.openshift.io
-export KEYCLOAK_SAAS_URL=https://sso.openshift.io/
-export OPENSHIFT_API_URL=https://api.starter-us-east-2.openshift.com
+* Execute:
+```bash
+$ mvn clean install
 ```
         
-Run the Integration Tests, Optionally Building
-----------------------------------------------
-
-* You need to set the following environment variables before running the integration tests: 
-
-		$ export LAUNCHER_MISSIONCONTROL_SERVICE_HOST=localhost
-		$ export LAUNCHER_MISSIONCONTROL_SERVICE_PORT=8080
+Build and Run the Unit and Integration Tests
+--------------------------------------------
 
+* First source the [the environment script](README.md#setting-up-the-environment)
 
 * To build the project and run the integration tests, allowing Maven to start the WildFly Swarm server:
- 
-        $ mvn clean install -Pit
-
+```bash
+$ mvn clean install -Pit
+```
 
 * To skip building and just run the integration tests, allowing Maven to start the WildFly Swarm server:
-
-        $ mvn integration-test -Pit
+```bash
+$ mvn integration-test -Pit
+```
 
 Reindex the booster catalog
 ---------------------------
 
 Run the following command, replace TOKEN with the value defined in the environment variable `LAUNCHER_BACKEND_CATALOG_REINDEX_TOKEN`. Doesn't need to be specified if the environment variable doesn't exist in the running environment:
 
         $ curl -v -H "Content-Type: application/json" -d '{}' -X POST  https://localhost:8180/api/launchpad/catalog/reindex\?token\=TOKEN
+        
+        
+Setup git providers default credentials (No KeyCloak mode)
+----------------------------------------------------------
+
```

Comment:
I think we need to make it more visually obvious here that there are 3 different options. Right now it just looks like a list of 6 items and that you have to perform all of them.

Reply:
Also add a text perhaps that says that you can use any combination of them (as long as you have at least one configured).

Reply:
For now, I don't think we can use any of them, only github is plugged. It's more for tests (for now)